### PR TITLE
Presets: hack-fix for multiple select dropdowns with 199+ elements

### PR DESCRIPTION
**Pull request by limonspb**
The number of keywords for presets reached 199 and revealed a bug in multiple select preventing it from scrolling.

Fixes:
https://github.com/betaflight/betaflight-configurator/issues/2789

the issue caused by the bug in multiple select jquery component. Described here:
https://github.com/wenzhixin/multiple-select/issues/552

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]
😄 @limonspb we need to just rewrite this thing with vue finally and then write tests for this upfront 😉

**Review by Asizon — CHANGES_REQUESTED** [review]
I dont think that we need a timer here, we can made a better workarround thought.

**Comment by limonspb** [comment]
> 😄 @limonspb we need to just rewrite this thing with vue finally and then write tests for this upfront 😉

Because vue third party components can't have bugs? Lol

**Comment by haslinghuis** [comment]
@limonspb there is a better workaround in the attached link.
```
$(function() {
   $('select').multipleSelect();
  
   // here I'm revealing the container that has the dropdowns
   // and after it's done showing, I refresh the multipleSelect
   $('#btn').click(() => $('#wrapper').show(function() {
      $('select').multipleSelect('refresh');	
   }));
})
```

**Comment by limonspb** [comment]
> @limonspb there is a better workaround in the attached link.
> 
> ```
> $(function() {
>    $('select').multipleSelect();
>   
>    // here I'm revealing the container that has the dropdowns
>    // and after it's done showing, I refresh the multipleSelect
>    $('#btn').click(() => $('#wrapper').show(function() {
>       $('select').multipleSelect('refresh');	
>    }));
> })
> ```

This is basically the same solution. They are using animation of toggle like setTimout(). Which is probably even a little more confusing

**Comment by haslinghuis** [comment]
You could add a comment for maintenance and clarity:  `// TODO: Remove workaround if fixed in wenzhixin/multiple-select#552`

**Comment by haslinghuis** [comment]
Point is we should avoid using a timeout like @Asizon said.

**Comment by limonspb** [comment]
> Point is we should avoid using a timeout like @Asizon said.

That does not mean we should hide timeout behind the fake animations

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
When you guys agree, we can move ahead ;-)

**Comment by Asizon** [comment]
@ctzsnooze there is nothing to agree, we are looking for a better solution :)

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
